File: flex/relay/relay.c
#include <wiringPi.h>
#include <stdio.h>

//Pin 29 on Raspberry Pi corresponds to BCM GPIO 5 and wiringPi pin 21
#define RelayPin 21

int main(void)
{
  if(-1 == wiringPiSetup())
  {
      printf("setup wiringPi failed!");
      return 1;
  }

  //Set GPIO pin
  pinMode(RelayPin, OUTPUT);

  //Turn on the relay
  digitalWrite(RelayPin, HIGH);
  //Wait for 3 seconds
  delay(3000);
  //Turn off the relay
  digitalWrite(RelayPin, LOW);

  return 0;
}

File: flex/relay/python/relay.py
# coding=utf-8
import time

import RPi.GPIO as GPIO

# Relay - pin 29 on Raspberry Pi corresponds to BCM GPIO 5
RelayPin = 29

# to use Raspberry Pi board pin numbers
GPIO.setmode(GPIO.BOARD)
GPIO.setwarnings(False)

# set up GPIO output channel
GPIO.setup(RelayPin, GPIO.OUT)

GPIO.output(RelayPin, GPIO.HIGH)
time.sleep(3)
GPIO.output(RelayPin, GPIO.LOW)
GPIO.cleanup()
